persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/util/CacheMapUtil.java

```diff
@@ -89,11 +89,17 @@
     }
 
     @PostConstruct
-    private void initializeCacheMemory() {
+    private void init() {
+        initializeCacheMemory();
+    }
+
+    public void initializeCacheMemory() {
+        
         // CHANGES TO THIS LIST MUST BE PROPAGATED TO 'GlobalProperties'
         this.cacheEnabled = (!authenticate.isEmpty() 
```

Comment:
I know you didn't change this, but is it even possible for `authenticate` to be empty? The annotation is `@Value("${authenticate:false}")`, so I think this is always true.

Reply:
I think in theory this would be possible when portal.properties contains this:

```
authenticate=
```

Spring will interpret this as `authenticate` being intialized to empty string.